packages/chakra-components/src/components/Election/Questions/Form.tsx

```diff
@@ -1,15 +1,18 @@
 import { Wallet } from '@ethersproject/wallet'
 import { useElection } from '@vocdoni/react-providers'
 import { ElectionResultsTypeNames, PublishedElection } from '@vocdoni/sdk'
-import React, { createContext, PropsWithChildren, ReactNode, useContext, useEffect } from 'react'
+import React, { createContext, PropsWithChildren, ReactNode, useContext, useEffect, useMemo, useState } from 'react'
 import { FieldValues, FormProvider, useForm, UseFormReturn } from 'react-hook-form'
 import { useConfirm } from '../../layout'
-import { QuestionsConfirmation } from './Confirmation'
+import { MultiElectionConfirmation } from './MultiElectionConfirmation'
+import { ElectionStateStorage, RenderWith, SubElectionState, SubmitFormValidation } from './Questions'
+
+export const DefaultElectionFormId = 'election-questions'
```

Comment:
Note it had the `election.id` appended because, in case you want to render multiple elections in the same page by simply adding multiple electionproviders, it would collide having the same id for all of them. The default election form id should still have a unique identifier to avoid this collision.

Reply:
This was a problem before, when MultiElection logic was splitted with the SingleElection logic. I won't explain why because it won´t be a problem anymore, so I'm going to restore the previous logic where the form id contains the root election id. 

Anyway I will let the possibility of adding the form id manually, so will be more extensible.

Reply:
4fd42c6